server/public/landing/landing.controller.js

```diff
@@ -1,30 +1,30 @@
-(function () {
+;(function () {
   'use strict'
   angular
     .module('jubilant-umbrella.landing')
     .controller('LandingController', LandingController)
 
-  function LandingController ($state) {
+  function LandingController (Authentication, $state) {
     // Initialization
     var vm = this
 
     // Variables
-    var email = null
-    var password = null
+    vm.email
```

Comment:
This is the school sign in, correct? If so, schools don't have an email or phone number.